ghost/admin/app/components/gh-post-settings-menu.hbs

```diff
@@ -138,15 +138,6 @@
                         </button>
                         {{svg-jar "arrow-right"}}
                     </li>
-                    {{#if (and this.post.isPost this.showEmailNewsletter (not-eq this.settings.membersSignupAccess "none") (not-eq this.settings.editorDefaultEmailRecipients "disabled"))}}
-                        <li class="nav-list-item">
-                            <button type="button" {{action "showSubview" "email-settings"}} data-test-button="email-settings">
-                                <b>Email newsletter</b>
-                                <span>Customize email settings</span>
-                            </button>
-                            {{svg-jar "arrow-right"}}
-                        </li>
-                    {{/if}}
```

Comment:
If the email newsletter subview is no longer available it would be good to clean up the view and conditionals for it as well as this link.

Reply:
Fixed :)